### ISSUE TITLE
App crash on Master Password re-prompt when viewing/editing Card

### ISSUE THREAD

**Issue by informagico** [bitwarden/mobile#2791]
### Steps To Reproduce

1. Try to view/edit a card with re-prompt Master Password enabled
2. Enter Master Password
3. Confirm
4. App crashes

### Expected Result

View or enter edit-mode

### Actual Result

App crashes

### Additional Context

Of course I am using up-to-date Google Play version

### Operating System

Android

### Operating System Version

Android 13 (September update)

### Device

Google Pixel 7

### Build Version

TQ3A.230901.001

### Beta

- [ ] Using a pre-release version of the application.

**Comment by cloudishBenne** [comment]
Same here. Now i can't use my government login method...

**Comment by PixelEnvision** [comment]
Related? #2478

**Comment by cloudishBenne** [comment]
Nope, but logging in with master password instead of biometrics solved this.

**Comment by informagico** [comment]
@cloudishBenne can confirm! I'll update the issue with this detail.

Isn't there any option to use biometrics instead of master password when re-prompted? I'm sure I've seen this in some future update/idea. Didn't I?

**Comment by IrisV3rm** [comment]
> Nope, but logging in with master password instead of biometrics solved this.

This also seemed to work for me!
